### PR TITLE
 update dependencies

### PR DESCRIPTION
**Pull request by drh-stanford**
This PR updates our gem requirements to pull in the latest resque (1.26.0) and correct bluepill version (sadly from their master branch).

Also, the rubocop upgrade caused a few changes, as did the code climate stuff.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-CodeClimate::TestReporter.start
+require 'simplecov'
+SimpleCov.start
+
 require 'coveralls'
 Coveralls.wear!
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
-  CodeClimate::TestReporter::Formatter,
-  Coveralls::SimpleCov::Formatter
-]
```
Comment (ndushay) [inline spec/spec_helper.rb]: See https://docs.codeclimate.com/docs/test-coverage-troubleshooting-tips#section--send-coverage-to-both-code-climate-and-coveralls-ruby-only-

...

Comment (drh-stanford) [inline spec/spec_helper.rb]: Yes, I saw that but `CodeClimate` was not defined -- the latest version of codeclimate says it's all built-in now. See https://github.com/codeclimate/ruby-test-reporter/blob/master/CHANGELOG.md#v100-2016-11-03 and https://docs.codeclimate.com/docs/ruby

Honestly after looking at this for a few minutes it's not clear at all what the current practice is. Let me see if I can find an example in sul-dlss...

Comment (drh-stanford) [inline spec/spec_helper.rb]: Ok, Stacks is doing just the `SimpleCov.start` way now. See https://github.com/sul-dlss/stacks/commit/0c9a534548f44f83fb1602ba251303a37507b3a4

**Comment by drh-stanford** [comment]
I'm going to redo this PR

File: spec/spec_helper.rb (hunk 1 above)
Comment (ndushay) [inline spec/spec_helper.rb]: kewl.  Good to know.

**Pull request closed without merging**